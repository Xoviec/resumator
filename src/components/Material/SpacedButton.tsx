import { Button } from "@mui/material";
import { styled } from "@mui/material/styles";
import { spacing } from "@mui/system";

export const SpacedButton = styled(Button)(spacing);
