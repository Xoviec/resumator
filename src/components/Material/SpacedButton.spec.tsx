import { render } from "@testing-library/react";
import { SpacedButton } from "./SpacedButton";

describe("SpacedButton", () => {
  it("should render correctly", () => {
    const { asFragment } = render(<SpacedButton />);

    expect(asFragment()).toMatchSnapshot();
  });
});
