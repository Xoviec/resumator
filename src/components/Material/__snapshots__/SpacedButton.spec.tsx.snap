// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpacedButton should render correctly 1`] = `
<DocumentFragment>
  <button
    class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root css-1o1c5h5-MuiButtonBase-root-MuiButton-root"
    tabindex="0"
    type="button"
  >
    <span
      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
    />
  </button>
</DocumentFragment>
`;
